CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## TBD
+
+* Report internal errors when serialization fails
+  [#581](https://github.com/bugsnag/bugsnag-android/pull/581)
+
+* Buffer io when reading from cached error file
```

Comment:
This should say what this is potentially fixing. Possible crash from running out of memory? Unbounded IO? etc

Reply:
Updated to clarify that from the end-user's perspective this will improve the SDK's performance.